jmxtrans/pom.xml

```diff
@@ -354,8 +350,7 @@
 									fi</script>
 							</preremoveScriptlet>
 							<requires>
-								<require>java</require>
-								<require>/usr/bin/lsb_release</require>
+								<require>java >= 1.7</require>
```

Comment:
Sounds good.  The jdk7 requirement is why I didn't add images for older debian/centos releases that didn't have OOTB packages for jdk7 (debian squeeze for example).